File: langgraph/data/equiv/87.py
def find_coordinates(matrix, target):
    positions = []
    for row_index, row in enumerate(matrix):
        for col_index, value in enumerate(row):
            if value == target:
                positions.append((row_index, col_index))

    sorted_positions = sorted(positions, key=lambda pos: pos[1], reverse=True)
    final_sorted_positions = sorted(sorted_positions, key=lambda pos: pos[0])
    return final_sorted_positions

# method name: get_row -> find_coordinates
# parameter name: lst -> matrix, x -> target
# list comprehension replaced with nested for loops
# variable name: coords -> positions
# split sorting into two steps for clarity

File: langgraph/result/qwen/32b-eq2js/87.js
function findCoordinates(matrix, target) {
    const positions = [];
    for (let rowIndex = 0; rowIndex < matrix.length; rowIndex++) {
        const row = matrix[rowIndex];
        for (let colIndex = 0; colIndex < row.length; colIndex++) {
            const value = row[colIndex];
            if (value === target) {
                positions.push([rowIndex, colIndex]);
            }
        }
    }

    const sortedPositions = positions.sort((a, b) => b[1] - a[1]);
    const finalSortedPositions = sortedPositions.sort((a, b) => a[0] - b[0]);
    return finalSortedPositions;
}